### ISSUE TITLE
Create pull request templates

### ISSUE THREAD

**Issue by kevtan** [stanford-ssi/sequoia-software#13]
It would be helpful to create pull request templates to improve and standardize the contributing experience. I'd like to open up the discussion here to see what kinds of issues everyone would like to see. From my time at Facebook, here are the biggest sections that jump out to me:

1. Summary
2. Test plan

@lgnashold @polygnomial Are there any other sections you'd like to see on pull requests so that reviewers have enough context to merge changes?

**Comment by polygnomial** [comment]
Yep, also want to see all **requirements** that this feature covers. We can discuss more today in the software breakout room.

**Comment by polygnomial** [comment]
Was there any discussion on Sunday? (do not recall any from my short time in the software breakout room)
